src/helper/db.js

```diff
@@ -0,0 +1,37 @@
+const low = require('lowdb')
+const FileSync = require('lowdb/adapters/FileSync')
+const adapter = new FileSync('db.json')
```

Comment:
Maybe smart to specify the path a little bit more clearly. I don't know if this is relative to this file, to the root of the project, etc. Maybe something like `path.resolve(__dirname, '../../db.json')`